samples/nrf9160/lwm2m_client/src/lwm2m/lwm2m_location.c

```diff
@@ -22,21 +22,21 @@ static bool event_handler(const struct event_header *eh)
 	if (is_gps_pvt_event(eh)) {
 		struct gps_pvt_event *event = cast_gps_pvt_event(eh);
 
-		float32_value_t latitude = double_to_float32(event->pvt.latitude);
-		float32_value_t longitude = double_to_float32(event->pvt.longitude);
-		float32_value_t altitude = double_to_float32(event->pvt.altitude);
-		float32_value_t speed = double_to_float32(event->pvt.speed);
-		float32_value_t radius = double_to_float32(event->pvt.accuracy);
-
-		lwm2m_engine_set_float32(LWM2M_PATH(LWM2M_OBJECT_LOCATION_ID, 0, LATITUDE_RID),
-					 &latitude);
-		lwm2m_engine_set_float32(LWM2M_PATH(LWM2M_OBJECT_LOCATION_ID, 0, LONGITUDE_RID),
-					 &longitude);
-		lwm2m_engine_set_float32(LWM2M_PATH(LWM2M_OBJECT_LOCATION_ID, 0, ALTITUDE_RID),
-					 &altitude);
-		lwm2m_engine_set_float32(
+		double latitude = (double)event->pvt.latitude;
```

Comment:
No need for explicit cast operations - struct [gps_pvt](https://github.com/nrfconnect/sdk-nrf/blob/50f05db985f701e0cbe877c3e1c41c7451df2774/include/drivers/gps.h#L71)

Reply:
True, since the PR has been merged into the upmerge branch I suggest we clean this up after the upmerge to avoid noise.